sdk/eventhub/event-hubs/samples/runsamples.ts

```diff
@@ -0,0 +1,28 @@
+import * as dotenv from "dotenv";
+import process from "process";
```

Comment:
Since `process` is a global in node.js, it shouldn't need to be imported. I think bundlers also know how to shim the global if you're concerned about running the samples in browsers, but I think you'd potentially want to depend on the `process` module as well then.

Reply:
Ah, it wasn't intentional and I'm totally happy to just remove it.

Reply:
It's actually better to explicitly import generally speaking. It gives bundlers a static syntactic pattern to replace. Replacing globals is more error prone and in some bundlers needs more configuration (e.g. `inject` for rollup).

Reply:
Ah, didn't know about the extra configuration needed, I'm assuming even for node bundles. Then yeah, I take back my comment. Since these samples also aren't meant to run in the browser as is I don't think you need to depend on the `process` module either.

Reply:
It's removed for now but that was an interesting point :)